src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1123,10 +1123,10 @@ public Integer getContingencyCount(UUID studyUuid, List<String> contingencyListN
         return actionsService.getContingencyCount(networkuuid, variantId, contingencyListNames);
     }
 
+    // TO DELETE
```

Comment:
So, remove this method, which is not used anymore

Reply:
done